src/API calls/Spotify/spotifyAPI.js

```diff
@@ -0,0 +1,26 @@
+import spotifyConfig from "../../Config/spotify";
```

Comment:
Spaces in the path name are generally not recommended. I'd change it to something like `Services/spotify/` (lowercase naming for folders unless its a component name or the folder is directly in `src`)

Reply:
I don't really get what you mean as this is the config file for our API endpoint and I thought this one should be used with absolute path? Could you elaborate please? :)

Reply:
Ah, sorry, it wasn't for this line of code, but for the folder this file is in ("API calls")